### ISSUE TITLE
Repeated hotkeys while keeping modifier pressed

### ISSUE THREAD

**Issue by linuz90** [JohannesKlauss/react-hotkeys-hook#898]
**Is your feature request related to a problem? Please describe.**
I'm implementing a Cmd+L hotkey to toggle a sidebar, and I think that while **keeping Cmd pressed**, I should be able to **press L repeatedly** to toggle the sidebar. I haven't found a way to create this behavior with the library.

**Additional context**
I think this is a common behavior on native OS hotkeys, so I hope an option to add this behavior can be added.

Another example I can't implement right now: Cmd+1, 2, 3... to change "room" in my webapp. After going to a room while keeping Cme pressed, I can't go back to it. It's like once a hotkey with modifier is pressed, it can't be triggered again without lifting all fingers.

This is my current implementation:

```
export default function useAppHotkeys() {
  const commandKey = navigator.platform.includes("Mac") ? "meta" : "ctrl";

  // Cmd + L to trigger Sidebar
  const setSidebar = useSetRecoilState(sidebarState);
  const toggleSidebar = () => setSidebar((prev) => ({ ...prev, open: !prev.open }));
  useHotkeys(`${commandKey}+l`, toggleSidebar, { ...defaultOptions }, [setSidebar]);

  // Cmd + 1, 2, 3... to change story room
  const setRoom = useSetRecoilState(currentStoryRoom);
  useHotkeys(`${commandKey}+1`, () => setRoom(0), { ...defaultOptions }, [setRoom]);
  useHotkeys(`${commandKey}+2`, () => setRoom(1), { ...defaultOptions }, [setRoom]);
  useHotkeys(`${commandKey}+3`, () => setRoom(2), { ...defaultOptions }, [setRoom]);
  useHotkeys(`${commandKey}+4`, () => setRoom(3), { ...defaultOptions }, [setRoom]);
  useHotkeys(`${commandKey}+5`, () => setRoom(4), { ...defaultOptions }, [setRoom]);
  useHotkeys(`${commandKey}+6`, () => setRoom(5), { ...defaultOptions }, [setRoom]);
}

const defaultOptions: Options = {
  enableOnFormTags: ["INPUT", "TEXTAREA", "SELECT"],
};
```

**Comment by ngyikp** [comment]
Having the same issue, 'Meta' on Mac doesn't repeat when you hold down the Command key, other modifiers like Ctrl/Alt/Shift are fine

https://codesandbox.io/s/flamboyant-pare-hdrti5

**Comment by FullstackJack** [comment]
Yes, this is definitely a bug. I think it has something to do with the keys getting unregistered after the event, but there should be logic to keep modifier keys from deactivating until an actual key up. I'd be interested in working on a fix, but I've only started using the library today. Of course, I'm implementing an undo/redo functionality in my app and this issue was immediately apparent since I hold the modifier (meta a.k.a. cmd) while pressing Z.

**Comment by animify** [comment]
+1 to this, encountering it now trying to use the lib for our company

**Comment by animify** [comment]
Perfect @JohannesKlauss lemme know if I can help or accelerate it in any way 🙇

**Comment by JohannesKlauss** [comment]
Feel free to create a pr for this

**Comment by JohannesKlauss** [comment]
Will be fixed in 4.3.3
